### DISCUSSION TITLE
Building examples for stm32 discovery board

### DISCUSSION THREAD

**Discussion by wallek876** [hathach/tinyusb#1652] [Q&A]
I've been trying to build the examples in tinyusb for the stm32f769i-discovery board.

After following the steps described in the "Getting Started" page, I am getting an error that don't know how to handle, when I run inside an example folder the command:

```
make BOARD=stm32f769disco PORT=1 all
```

I get the following output:

```
"Using OTG_HS in HighSpeed mode"
AS startup_stm32f769xx_asm.o
cc1.exe: fatal error: /cygdrive/c/git/tinyusb/hw/mcu/st/cmsis_device_f7/Source/Templates/gcc/startup_stm32f769xx.s: No such file or directory
compilation terminated.
make: *** [../../rules.mk:137: _build/stm32f769disco/obj/hw/mcu/st/cmsis_device_f7/Source/Templates/gcc/startup_stm32f769xx_asm.o] Error 1
```

I've looked inside the `tinyusb/hw/mcu/st/cmsis_device_f7/Source/Templates/gcc/` and there is a file there called `startup_stm32f769xx.s`, I don't know how to proceed from this point, or does the error refer to some other missing file?

**Comment by hathach** [comment]
this is due to cygwin file management, to be honest, I don't use windows to know how to fix this. Maybe other could help, or try to search discussion, there is a few posts mentioning compiling with windows.

**Comment by kkitayam** [comment]
If you use [Arm GNU Toolchain](https://developer.arm.com/downloads/-/arm-gnu-toolchain-downloads) on windows, I recommend msys2 or cmd.exe instead of cygwin. See also #669.
